wpilibcExamples/build_cpp_examples.bzl

```diff
@@ -0,0 +1,150 @@
+load("@wpi_bazel_rules//rules:cc.bzl", "wpilib_cc_library")
+load("@wpi_bazel_rules//rules:halsim_binary.bzl", "wpilib_cc_halsim_binary")
+
+EXAMPLE_FOLDERS = [
```

Comment:
Hmm, is it possible to read the existing JSON manifest instead of listing all these here?

Reply:
I should have clarified I'm by no means an expert on what bazel can do. But, from what I understand, I don't think so. I don't think it can read non-build files during the loading phase, and the building phase cannot declare new build targets.